homeassistant/components/growatt_server/strings.json

```
@@ -425,5 +425,111 @@
         "name": "Maximum power"
       }
     }
+  },
+  "services": {
+    "set_grid_first": {
+      "name": "Set grid first",
+      "description": "Set the inverter to export energy to the grid",
+      "fields": {
+        "discharge_power_rate": {
+          "name": "Discharge Power Rate",
```

Sentence-casing here and on several strings below (I did not mark them all):

```suggestion
          "name": "Discharge power rate",
```